packages/e2e-test-utils/src/ensure-sidebar-opened.js

```diff
@@ -4,14 +4,11 @@
  * @return {Promise} Promise resolving once the edit post sidebar is opened.
  */
 export async function ensureSidebarOpened() {
-	// This try/catch flow relies on the fact that `page.$eval` throws an error
-	// if the element matching the given selector does not exist. Thus, if an
-	// error is thrown, it can be inferred that the sidebar is not opened.
-	try {
-		return page.$eval( '.edit-post-sidebar', () => {} );
-	} catch ( error ) {
-		return page.click(
-			'.edit-post-header__settings [aria-label="Settings"]'
-		);
+	const toggleSidebarButton = await page.$(
+		'.edit-post-header__settings [aria-label="Settings"][aria-expanded="false"]'
+	);
+
+	if ( toggleSidebarButton ) {
+		await toggleSidebarButton.click();
```

Comment:
The previous implementation wasn't working for some reason. This is a better way to do it anyway.